app/Http/Controllers/TaggingController.php

```diff
@@ -64,7 +64,7 @@ public function showGames($tagid){
         ]);
     }
 
-    public function store(Request $request){
+    public function store(Request $request) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->